docs/apis-clients/java-client/zeebe-test.md

```diff
@@ -0,0 +1,74 @@
+---
+id: zeebe-test
+title: "Zeebe test"
+---
+
+You can use the `zeebe-test` module to write JUnit tests for your job worker and BPMN process. This provides a JUnit rule to bootstrap the broker and some basic assertions.
```

Comment:
:x: If we target 1.4 as release date for this change, then I think this document is outdated. Please decide whether this PR is for now (and foreshadowing a new way to test things, or for 1.4 and replacing the old way to test things). Then change text accordingly.

Reply:
This PR will be for 1.4. I've discussed with Ole and his plan was to remove this page in 1.4 anyways. So instead of fixing this, I will remove it.

People who are using a lower version and are still using this should still be able to find it using an older version of the docs.